src/Sylius/Bundle/CoreBundle/Resources/config/app/sylius/sylius_product.yml

```diff
@@ -17,10 +17,6 @@ sylius_product:
         product_variant:
             classes:
                 model: Sylius\Component\Core\Model\ProductVariant
-                repository: Sylius\Bundle\ProductBundle\Doctrine\ORM\ProductVariantRepository
```

Comment:
Since variants have been merge with product is there any point in leaving the declaration only in the core, and not in the bundle itself?